cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -191,16 +194,25 @@ def test_vmware_vds_ui_tagging():
         testtype: integration
         testSteps:
             1.Integrate VMware provider in CFME
-            2.Compute > Infrastructure > Host > [Select host] > Properties > Network
+            2.Compute > Infrastructure > Networkiong
             3.Check if host has Distributed Switch and it is displayed on this page
             4.If displayed, try to select Policy->Assign Tag to DSwitch.
         expectedResults:
             1.
-            2.Properties page for the host opens.
+            2.Networking Page opens
             3.If DSwitch exists it will be displayed on this page.
             4.You can assign tags to DSwitch.
     """
-    pass
+    switches_collection = appliance.collections.infra_switches
+    switches = [switch for switch in switches_collection.all() if switch.name == 'DSwitch']
+    assert not switches == [], "There are no DSwitches on Networking Page"
```

Comment:
This syntax is clunky, and can simply be `assert switches, 'message'`, as an empty list is falsy